File: Domain/License.cs
using System;
using System.Collections.Generic;

namespace Domain
{
    public class License
    {
        public Guid Id { get; set; }
        public string Name { get; set; }
        public string DisplayName { get; set; }

        public Guid ModalityId { get; set; }
        public Modality Modality { get; set; }

        public ICollection<TechnologistLicense> Technologists { get; set; }

    }
}

File: client-app/src/app/models/license.ts
export interface ILicense {
  id: string;
  name: string;
  displayName: string;
  modalityId: string;
}

export class LicenseFormValues {
  id?: string = undefined;
  name: string = "";
  displayName: string = "";
  modalityId: string = "";
}
